native/exe/SageLauncher/launcher.cpp

```diff
@@ -625,8 +625,8 @@ int WINAPI WinMain( HINSTANCE hInst, 	/*Win32 entry-point routine */
 	// Fix the current working directory to be where the EXE is located
 	LPTSTR folderPath = new TCHAR[2048];
 	GetModuleFileName(NULL, folderPath, 2048);
-	int appLen = strlen(folderPath);
-	for (int i = appLen - 1; i > 0; i--)
+	size_t appLen = strlen(folderPath);
+	for (size_t i = appLen - 1; i > 0; i--)

```

Comment:
Again, dangerous if appLen is zero (which I know shouldn't happen, but still...it's good coding).

Reply:
Ditto fix